.github/workflows/update-collections.yml

```diff
@@ -0,0 +1,45 @@
+# Triggers on every release and triggers `sync-collections` scripts by sending the added, updated and removed
+# collections ids
+name: Update Collections
+
+on:
+  release:
+    types:
+      - published
+
+env:
+  node_version: "16.x"
+
+jobs:
+  update-collections:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./firebase
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      # Setup Node.js with version `env.node_version`.
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.node_version }}
+          # cache: "npm"
+          # cache-dependency-path: "**/package-lock.json"
```

Comment:
any reason for these comments?

Reply:
Sorry about that, I forgot to uncomment. I'm currently using [`act`](https://github.com/nektos/act) to test the actions locally instead testing in Github, and it has some specific limitations (NPM cache is one of them...)